infrastructure/sentry-subgraph/src/referee.ts

```diff
@@ -404,16 +446,11 @@ export function handleBatchRewardsClaimed(event: BatchRewardsClaimedEvent): void
       log.warning("Failed to find refereeConfig handleBatchRewardsClaimed TX: " + event.transaction.hash.toHexString(), [])
       return;
     }
-    
-    // Set the referee version loaded from the graph
-    const refereeVersion = refereeConfig.version;
 
     const ownerWallet = SentryWallet.load(sentryKey.sentryWallet)
     if (ownerWallet) {
       if (
-        // Require KYC approval for all versions except 6 and below
-        // DO NOT REMOVE THIS CHECK or the graph numbers will be incorrect
-        (ownerWallet.isKYCApproved || refereeVersion.gt(BigInt.fromI32(6))) &&
+        ownerWallet.isKYCApproved &&
```

Comment:
Why was referee version check here removed?

This is required to allow non-KYC claims to go through in the graph on new referee versions.

Reply:
This is not needed with adding the new V2 events as we are not using these handlers for anything above Referee version 6

```
if (refereeConfig.version.gt(BigInt.fromI32(6))) {
  // Event replaced in newer versions of the Referee for simpler event handlers
  return;
}
```